api/src/pcapi/routes/serialization/public_api_collective_offers_serialize.py

```diff
@@ -1,5 +1,117 @@
+from datetime import datetime
+from datetime import timezone
+import enum
+from typing import Any
+
+from pydantic import validator
+
 from pcapi.core.educational.models import CollectiveOffer
+from pcapi.core.educational.models import StudentLevels
 from pcapi.routes.serialization import BaseModel
+from pcapi.serialization.utils import to_camel
+from pcapi.validation.routes.offers import check_offer_name_length_is_valid
+
+
+class OfferAddressType(enum.Enum):
+    OFFERER_VENUE = "MY_ADDRESS"
```

Comment:
```suggestion
    OFFERER_VENUE = "MY_VENUE_ADDRESS"
```
? comme il y a potentiellement plusieurs lieux c'est peut-être + explicite :)

Reply:
malheureusement c'est dans la spec